src/components/Search/Search.component.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Styled from './styled';
+
+const Search = () => (
+  <Styled.SearchContainer>
```

Comment:
how could this be cleaner?

1. Use restructuring at import statement
2. Use static path `d` prop as constant in a separate file